digitaltwin_client/samples/readme.md

```diff
@@ -4,24 +4,79 @@ This directory contains samples demonstrating creating and using Digital Twin in
-* [digitaltwin_sample_environmental_sensor](./digitaltwin_sample_environmental_sensor) library that implements a sample environmental sensor Digital Twin interface.  This interface demonstrates all concepts of the implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.
+### Executable Source Code
+* [digitaltwin\_sample\_device](./digitaltwin_sample_device) - The executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation of interfaces and basic operation when using Iot Hub.
+* [digitaltwin\_sample\_ll_device](./digitaltwin_sample_ll_device) - The executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation interfaces and basic operation.  It uses the lower level (\_LL\_) layer and enables the Device Provisioning Service (DPS) to connect to Iot Central. It is appropriate for devices with limited resources.
+
+### Libraries
+* [digitaltwin\_sample\_device_info](./digitaltwin_sample_device_info) - Library that implements a sample device info Digital Twin interface of helper functions.  This interface reports information about the device - such as OS version, amount of storage, etc.
+* [digitaltwin\_sample\_environmental_sensor](./digitaltwin_sample_environmental_sensor) - Library that implements a sample environmental sensor Digital Twin interface of helper functions.  This interface demonstrates all concepts of implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.
 
 ## Getting started
+Build the Digital Twin Client SDK, if not already completed.  Follow instructions [here](../doc/building_sdk.md).
+
+### digitaltwin\_sample\_device
+
+* Use cmake to build the [digitaltwin\_sample\_device](digitaltwin_sample_device) directory.
+
+  From the azure-iot-c-sdk repository root:
+  ```
+  cd cmake
+  cmake --build .
+  ```
+  This will build the `digitaltwin_sample_device` executable along with the aforementioned sample interfaces it uses.
+
+* Get the Connection String of the Iot Hub device you wish to DigitalTwin enable.  
+
+  This can be found via the [azure portal](https://portal.azure.com):  
+    1. Go to your IoT Hub
+    2. Under Explorers in the left pane, select IoT Devices
+    3. Under Device ID, select a device.  Else, add a new device and then select it.
+    4. Copy the Primary Connection String
+
+* Run the sample.
+
+  **If using Windows:** 
+
+  From the cmake directory:
+  ```
+  cd .\digitaltwin_client\samples\digitaltwin_sample_device\Debug\
+  ```
+  Run the executable using your connection string, e.g.:
+  ```
+  .\digitaltwin_sample_device.exe "HostName=yourHub;DeviceId=yourDigitalTwinDeviceId;SharedAccessKey=secret"
+  ```
+
+  **If using Linux:**
```

Comment:
I wonder if it warrants specifying here which version of Linux is supported.  I tested in Ubuntu 18.04.  The "Compiling the C Device SDK" instructions for Linux on a prior connected page is only for Ubuntu.  If so, should it also specify which version of Windows is supported?